css-four-column-layout/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  width: 1400px;
+  font-family: "Barlow", Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+}
+
+h2 {
+  text-align: center;
+  font-family: "Barlow", Arial, Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+h3 {
+  font-weight: 700;
+  text-align: center;
+}
+
+p {
+  font-weight: 400;
+  text-align: center;
+}
+
+img {
+  max-width: 90%;
+  max-height: 90%;
+}
+
+.icon {
+  max-width: 20%;
+  max-height: 20%;
+  margin-left: 36%;
+  margin-right: 36%;
```

Comment:
This is a lot of margin. Rather than using margin for this, I would recommend finding a different way to center this.